pkg/hubble/exporter/exporter.go

```diff
@@ -140,5 +144,19 @@ func (e *exporter) OnDecodedEvent(_ context.Context, ev *v1.Event) (bool, error)
 	if res == nil {
 		return false, nil
 	}
-	return false, e.encoder.Encode(res)
+	// Check if we are logging directly
+	if e.opts.Path == "stdout" {
+		jsonData, err := json.Marshal(res)
+		if err != nil {
+			e.logger.WithError(err).Error("Failed to marshal event")
+			return false, err
+		}
+		e.logger.Info(string(jsonData))
```

Comment:
I think it'd be much easier to post-process in SIEM and logs databases if we log the JSON flow as a separate field. It should help for use-cases where hubble flows need to be processed or indexed separately from the rest of the logs (ie JSON parsed or else).

```suggestion
		e.logger.WithField("flow", string(jsonData)).Info("Recorded Hubble flow")
```

Reply:
Thanks four your suggestion! I changed it accordingly.

Reply:
Wouldn't it be better to print `res`directly? It would be more readable because less escaping chars.
Like that:
```
e.logger.WithField("flow", res).Info("Recorded Hubble flow")
```